llvm/test/MC/X86/apx/and-reloc.s

```diff
@@ -0,0 +1,38 @@
+// RUN: llvm-mc -triple x86_64-linux-gnu -filetype=obj %s | llvm-readobj -r - | FileCheck %s
+
+// CHECK:      Relocations [
+// CHECK-NEXT:   Section ({{[0-9]+}}) .rela.text {
+{evex}  andb $foo, %al                      // CHECK-NEXT:     R_X86_64_8
```

Comment:
There are many forms for each opcode, duplicated across many files. Is there a way to deduplicate the tests?

I'd suggest llvm-objdump -dr  (inline relocations) to test both the encoding and the relocations, possibly making `encopt.s` unneeded.